src/Components/Artist/ArtistCollectionsRail/ArtistCollectionsRail.tsx

```diff
@@ -54,9 +54,9 @@ export class ArtistCollectionsRail extends React.Component<
 
           <Carousel
             height={200}
-            settings={{
-              slidesToScroll: 1,
-              infinite: true,
+            flickitySettings={{
```

Comment:
If thinking in declarative terms, one shouldn't need to know that it's flickity underneith, and keep things general. By encoding `flickity` into the prop name we're leaking implementation details. Devs can refer to the inner component docs to know where to look for settings details.